src/hid_manager.rs

```diff
@@ -0,0 +1,42 @@
+use libc::c_void;
+use cf::{CFAllocatorRef, CFDictionaryRef, CFIndex};
+use types::IOOptionBits;
+use io_return::IOReturn;
+
+#[doc(hidden)]
+#[repr(C)]
+pub struct __IOHIDManager {
+    __private: c_void,
+}
+pub type IOHIDManagerRef = *const __IOHIDManager;
+
+#[doc(hidden)]
+#[repr(C)]
+pub struct __IOHIDDevice {
+    __private: c_void,
+}
+pub type IOHIDDeviceRef = *const __IOHIDDevice;
```

Comment:
This should probably be `*mut` as well. This is defined in a separate header file (`IOKit/hid/IOHIDBase.h`) than `IOHIDManagerRef` (`IOKit/hid/IOHIDManager.h`). There's a note under the ["contributing" section of the README](https://github.com/dcuddeback/iokit-sys#contributing) that explains the code organization:

> If you plan to submit a pull request, please note the structure of the code. There is one file for each header file in the IOKit framework. For example, `src/io_return.rs` contains the definitions from `IOKit/IOReturn.h`. The definitions in each file are more or less in the same order that they appear in the matching header file. Each file is then re-exported in the crate root, e.g., `pub use io_return::*`.

Would you be able to split the definitions to follow this convention? If you look at the existing files, most of them have a comment at the top saying which header file they correspond to. That makes it easier to find the C definitions for comparing to the Rust defintions.